docs/framework/docker/aspnetmvc.md

```diff
@@ -50,43 +45,39 @@ The finished application is located in the
 
 ## Prerequisites
```

Comment:
VS2015 is a prereq, judging from the screenshots.  Publish UI is different in VS 2017

directly above this line in a part you didn't change is a link with link text dotnet/core-docs that should be dotnet/docs